services/src/handlers/wms.rs

```diff
@@ -241,7 +241,10 @@ where
         })
         .await?;
 
-    let colorizer = Colorizer::rgba(); // TODO: create colorizer from request
+    let colorizer = match request.styles.strip_prefix("custom:") {
+        None => Colorizer::Rgba,
```

Comment:
Could you provide as default a greyscale linear colorizer in the range of the datatype?

For instance, if you have u8, provide a linear colorize black to white from 0 to 255.
And if you have i16, provide one from -32768 to 32767.

You could also add a TODO then to incorporate the NODATA value once we have it in our metadata.

Reply:
a8f76b1
acbbc59